angular/src/components/set-password.component.ts

```diff
@@ -57,6 +63,13 @@ export class SetPasswordComponent extends BaseChangePasswordComponent {
             }
         });
 
+        // Automatic Enrollment Detection
+        const org = await this.userService.getOrganizationByIdentifier(this.identifier);
+        this.orgId = org?.id;
+        const policyList = await this.policyService.getAll(PolicyType.ResetPassword);
+        const policyResult = this.policyService.getResetPasswordPolicyOptions(policyList, this.orgId);
+        this.resetPasswordAutoEnroll = policyResult[1] && policyResult[0].autoEnrollEnabled;
```

Comment:
`identifier` will still be null for non-SSO flows, right? If that is true, we should wrap this in a conditional to make sure we don't check a bunch of stuff we don't need to.

Reply:
At this time, I don't believe there is a non-SSO scenario, but you make a great point about defensive programming - will update.

Reply:
Might be fine as is. I was thinking this component had wider scope. Thanks Vince!

Reply:
Updated with null check.